### PR TITLE
Clear filtered results when input search is empty

### PR DESCRIPTION
**Pull request by Rahmon**
### Description of the Change

This change updates to clear filtered results when the input search is empty.

### Benefits

There is no need to click on the Search button to reset the results.

### Verification Process

1. Add the Brightcove block and edit it
2. Type in the search field and hit Enter to filter the results
3. Erase the search term and check if the results are reset

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: assets/js/src/views/toolbar.js [hunk 1]
```diff
@@ -118,11 +119,18 @@ var ToolbarView = BrightcoveView.extend(
...
-      }
-    },
+		enterHandler : function ( event ) {
+			if ( event.keyCode === 13 ) {
+				this.searchHandler( event );
+			}
+		},
+
+		handleEmptySearchInput : function ( event ) {
+			if (this.model.get( 'search' ) && !event.target.value) {
```
Comment (oscarssanchez) [inline assets/js/src/views/toolbar.js:129]: @Rahmon what are your thoughts on enforcing code style? I know the plugin has many inconsistencies but would be good in my opinion to start enforcing them

**Review by oscarssanchez — COMMENTED** [review]
@Rahmon what are your thoughts on enforcing code style? I know the plugin has many inconsistencies but would be good in my opinion to start enforcing them

**Comment by oscarssanchez** [comment]
Also, it looks like you would need to rebuild the assets with the latest from develop @Rahmon

**Review by oscarssanchez — APPROVED** [review]

File: assets/js/src/views/toolbar.js (hunk 1 above)
Comment (Rahmon) [reply assets/js/src/views/toolbar.js:129]: Thanks Oscar.

I'll implement it on another PR.

**Pull request merged**